### DISCUSSION TITLE
new to this

### DISCUSSION THREAD

**Discussion by Forestspiritfarmer** [zippity21/Stardew_Valley_Regression_Mod#124] [Q&A]
okay so i want to Jas to use the custom dialog my question is what do i have to to for her to use it?

Another question is dose the abdl dialogue break when I make a single player save into a online coop?

**Comment by GardensStarlit** [comment]
The first time you talk to either Jas or Vincent, you should get a prompt asking if you're okay with modded dialogue from them -- the default *should* be "on" if you somehow bypass this prompt, unless you answer "No"

I haven't personally checked, but according to others, the mod is very buggy and prone to crashing on multiplayer -- I wouldn't recommend it.
